src/FluentValidation.Tests/MessageFormatterTests.cs

```diff
@@ -55,5 +58,89 @@ public class MessageFormatterTests {
 
 			result.ShouldEqual("bar baz");
 		}
+
+		[Fact]
+		public void Format_property_value()
+		{
+			using (new TemporalCultureExchange())
+			{
+				string result = formatter
+				.AppendPropertyValue(123.45)
+				.BuildMessage("{PropertyValue:#.#}");
+
+				result.ShouldEqual("123.5");
+			}
+		}
+
+		[Fact]
+		public void Understands_numeric_formats()

```

Comment:
This test is currently failing, could you take a look at why that's happening?

Reply:
How do I get the details from your automated build and tests? It does not fail locally.

Reply:
Here is the test output for the tests that are failing:

Space before the percent sign. This failed under net452:
```
xUnit.net 00:00:10.0480979]     FluentValidation.Tests.MessageFormatterTests.Understands_numeric_formats [FAIL]
Failed   FluentValidation.Tests.MessageFormatterTests.Understands_numeric_formats
Error Message:
 Assert.Equal() Failure
Expected: ($123.457) ($123.4568) 1234.57 91.20% 123 1.052033e+003 1052.03 0123
Actual:     ($123.457) ($123.4568) 1234.57 91.20 % 123 1.052033e+003 1052.03 0123
```

Negative number/brackets/trailing zeros. Failed under both netcoreapp2 and netcoreapp2.1
```
[xUnit.net 00:00:08.2479321]     FluentValidation.Tests.MessageFormatterTests.Understands_numeric_formats [FAIL]
Failed   FluentValidation.Tests.MessageFormatterTests.Understands_numeric_formats
Error Message:
 Assert.Equal() Failure
Expected: ($123.457) ($123.4568) 1234.57 91.20% 123 1.052033e+003 1052.03 0123
Actual:   -$123.457 -$123.4568 1234.567 91.200% 123 1.052033e+003 1052.03 0123
```

If you're not seeing the failures locally I expect it's related to the locale. Your `TemporalCultureExchange` only set `CurrentCulture`, but doesn't change `CurrentUICulture`, which I expect is the issue. Switching to the existing `CultureScope` will probably solve this.